.pytool/CISettings.py

```diff
@@ -5,13 +5,23 @@
 ##
 import os
 import logging
+import sys
 from edk2toolext.environment import shell_environment
 from edk2toolext.invocables.edk2_ci_build import CiBuildSettingsManager
 from edk2toolext.invocables.edk2_ci_setup import CiSetupSettingsManager
 from edk2toolext.invocables.edk2_setup import SetupSettingsManager, RequiredSubmodule
 from edk2toolext.invocables.edk2_update import UpdateSettingsManager
 from edk2toolext.invocables.edk2_pr_eval import PrEvalSettingsManager
 from edk2toollib.utility_functions import GetHostInfo
+from pathlib import Path
```

Comment:
It's in audit mode while repos are baselined. I agree though that other repos should be filtered. I'll update the filter list as part of this check-in to exclude dependent repos.

Reply:
On second thought, as we discussed offline, let's keep the filters focused on the repos at hand for the moment to avoid missing CodeQL alerts that may only be present with certain build configurations.

We can add filters as needed after respective repos have had a pass through there issue PRs.